package_bag/routines.py

```diff
@@ -72,15 +73,13 @@ def unpack_rename(self, bag_path, tmp):
     def validate_metadata(self, bag_path):
         """Validates the bag-info.txt file against the bagit profile"""
         new_bag = bagit.Bag(bag_path)
-        if "BagIt-Profile-Identifier" not in new_bag.info:
-            raise TypeError("No BagIt Profile to validate against")
+        with open("/code/package_bag/zorya_bagit_profile.json", "r") as fp:
```

Comment:
Errrr this looks like path joining that should be done with `os.path.join` and is also an absolute path the the container, which will NOT work in a non-container deployment.